app/cdash/app/Model/Build.php

```diff
@@ -649,9 +642,8 @@ public function GetErrors(array $propertyFilters = [], $fetchStyle = PDO::FETCH_
      * Returns all failures (errors), including warnings, for current build
      *
      * @param int $fetchStyle
-     * @return array|bool
      */
-    public function GetFailures(array $propertyFilters = [], $fetchStyle = PDO::FETCH_ASSOC)
+    public function GetFailures(array $propertyFilters = [], $fetchStyle = PDO::FETCH_ASSOC) : array|bool
```

Comment:
Same here.  A parameter type declaration can also be added for `$fetchStyle`.
```suggestion
    public function GetFailures(array $propertyFilters = [], int $fetchStyle = PDO::FETCH_ASSOC) : array|false
```